docs/conf.py

```diff
@@ -14,7 +14,13 @@
 # -- Project information -----------------------------------------------------
 
 project = "Pyodide"
-copyright = "2019-2021, Pyodide contributors and Mozilla"
+copyright = "2019-2022, Pyodide contributors and Mozilla"
+pyodide_version = "0.21.0.dev0"
+
+if any([(dev_ver in pyodide_version) for dev_ver in ("dev", "a", "b", "rc")]):
+    CDN_URL = "https://cdn.jsdelivr.net/pyodide/dev/full/"
```

Comment:
Is this right? For instance for an alpha or beta release we would want to take the "else" block. I think you maybe just want `if "dev" in pyodide_version:`

Reply:
You are right. I thought that we don't deploy alpha versions to CDN but we do.